tests/torch/quantization/test_quantizer_propagation_graph.py

```diff
@@ -1071,6 +1075,87 @@ def test_merge_redundant_subsequent_quantizers_across_graph(self, model_graph_qp
 
         assert len(remaining_pqs) == len(ref_remaining_pq_positions)
 
+
+class TestUnifinedScaleTypeAfterMergeQuantizers:
+    @staticmethod
+    def get_merged_pq(qpsg: QPSG) -> PropagatingQuantizer:
```

Comment:
Better to return the QPSG instead of a single quantizer. You can get PQs from a QPSG by calling `collect_all_propagating_quantizers`, for instance.